addon.py

```
@@ -1373,6 +1387,311 @@ def import_generated_asset_fal_ai(self, request_id: str, name: str):
+        enabled = bpy.context.scene.blendermcp_use_hunyuan3d
+        if enabled:
+            if not bpy.context.scene.blendermcp_hunyuan3d_secret_id or not bpy.context.scene.blendermcp_hunyuan3d_secret_key:
+                return {
+                    "enabled": False, 
+                    "message": """Hunyuan3D integration is currently enabled, but SecretId or SecretKey is not given. To enable it:
+                                1. In the 3D Viewport, find the BlenderMCP panel in the sidebar (press N if hidden)
+                                2. Keep the 'Use Tecent Hunyuan 3D model generation' checkbox checked
+                                3. Choose the right plaform and fill in the SecretId and SecretKey
+                                4. Restart the connection to Claude"""
+                }
+            message = f"Hunyuan3D integration is enabled and ready to use."
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)